app/src/main/java/chat/rocket/android/layouthelper/extra_action/upload/FileUploadActionItem.java

```diff
@@ -0,0 +1,31 @@
+package chat.rocket.android.layouthelper.extra_action.upload;
+
+import android.content.Intent;
+
+import chat.rocket.android.R;
+
+public class FileUploadActionItem extends AbstractUploadActionItem {
+
+  @Override
+  public int getItemId() {
+    return 13;
```

Comment:
Is it an aleatory ID?

Reply:
Not quite. All attachment types have an id. It just adds sequentially. The first one picked, was chosen beforehand by the previous dev.